cves/kernel/CVE-2012-6545.yml

```diff
@@ -161,10 +154,14 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    It seems this was a minor memory leak discovered and fixed by a developer. 
+    The only written history on this are the git commits, and this email sent 
+    by the developer who fixed it: https://www.openwall.com/lists/oss-security/2013/03/05/13. 
```

Comment:
Just a reminder that this could result in a dead link in the future.

Reply:
Good notice, but I think I'll leave it in for now. There were a some dead links on the CVE site, but the alive ones were valuable.